run/GCClassic/createRunDir.sh

```diff
@@ -1045,7 +1045,7 @@ fi
 
 #--------------------------------------------------------------------
 # Copy sample restart file to run directory
-# Bash functions used here are from ../shared/setupForRestarts.sh
+# Bash functions used here are from ./setupForRestarts.sh
 #--------------------------------------------------------------------
```

Comment:
Have you also updated the restart file path in `./run/shared/download_data.yml`?

Reply:
No, the GCClassic TOMAS restart files are not updated here. We only add TOMAS restarts for GCHP.